### PR TITLE
Update readme.txt and changelog.md

### PR DESCRIPTION
**Pull request by Zodiac1978**
Add Requires PHP header
Add new FAQ entry about form plugins and registration page.
Fix code example
Fix link in FAQ
Add 2.9.3 changelog entries
Remove superfluous "English" header in changelog

### CODE REVIEW

**Comment by websupporter** [comment]
Looks good to me. Thank you @Zodiac1978 :heart:

**Comment by pfefferle** [comment]
This changes should also made to the CHANGELOG.md?!?

**Comment by Zodiac1978** [comment]
@pfefferle What do you mean with your comment?

This is added to changelog.md also:
https://github.com/pluginkollektiv/antispam-bee/pull/353/commits/ee8ab28ecd2e39ff6791f33a6ca710a32011e1a3

**Comment by pfefferle** [comment]
Ah, missed that... sorry :(

**Review by pfefferle — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -3,8 +3,9 @@
 * Tags:              anti-spam, antispam, block spam, comment, comments, comment spam, pingback, spam, spam filter, trackback, GDPR
 * Donate link:       https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=TD4AMD2D8EMZW
 * Requires at least: 4.5
-* Tested up to:      5.4
-* Stable tag:        2.9.2
+* Tested up to:      5.5
+* Requires PHP:      5.2
+* Stable tag:        2.9.3-beta
```
Comment (pfefferle) [inline readme.txt:8]: needs to be changed for final release

Comment (florianbrinkmann) [reply readme.txt]: Done

**Review by florianbrinkmann — APPROVED** [review]

**Pull request merged**